tests/unit_tests/blockchain_db.cpp

```diff
@@ -242,27 +242,31 @@ TYPED_TEST_CASE(BlockchainDBTest, implementations);
 
 TYPED_TEST(BlockchainDBTest, OpenAndClose)
 {
-  std::string fname(tmpnam(NULL));
+  boost::filesystem::path tempPath = boost::filesystem::temp_directory_path() / boost::filesystem::unique_path();
+  std::string dirPath = tempPath.string();
 
-  this->set_prefix(fname);
+  this->set_prefix(dirPath);
```

Comment:
No., not at all. Looking at the variable (fname), indicated it was a filename, not a directory until I traced the code all the way up. I figured it would be more descriptive to rename it to what it actually does. 

Your point is taken  though. The intent was not to spam at all, but make the code directly readable.